code/src/components/Start.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { game } from "../reducers/game";
+import { startGame } from "../reducers/game";
+
+export const Start = () => {
+  const dispatch = useDispatch();
+
+  const handleInputChange = (event) => {
+    dispatch(game.actions.setUsername(event.target.value));
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    dispatch(startGame());
+  };
+
+  return (
+    <>
+      <h1>It's a maze.... ing</h1>
+      <h2>¯\_(ツ)_/¯</h2>
+
+      {/* Remember to use a form tag and the onSubmit event to handle enter clicks AND button clicks in one go */}
```

Comment:
Maybe this comment was more for educational purposes instead of explaining the code?